src/components/Question.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Text from '../customs/CustomText';
+
+
+const Question: React.FC = (props) => (
+  <Text style={{ fontSize: 22, marginHorizontal: 30 }}>
+    Q. {props.children}
+  </Text>
+);
+
+export default Question;
```

Comment:
\n

Reply:
oh you want an extra line at end of file